test/files/gem005/pass-gem005-body.ifc

```diff
@@ -0,0 +1,42 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [CoordinationView]'),'2;1');
+FILE_NAME('','2023-09-20T19:11:03',(''),(''),'IfcOpenShell-v0.7.0-64d98f129','IfcOpenShell-v0.7.0-64d98f129','');
+FILE_SCHEMA(('IFC2X3'));
+ENDSEC;
+DATA;
+#1=IFCPERSON($,$,'',$,$,$,$,$);
+#2=IFCORGANIZATION($,'',$,$,$);
+#3=IFCPERSONANDORGANIZATION(#1,#2,$);
+#4=IFCAPPLICATION(#2,'v0.7.0-64d98f129','IfcOpenShell-v0.7.0-64d98f129','');
+#5=IFCOWNERHISTORY(#3,#4,$,.ADDED.,$,#3,#4,1695237063);
+#6=IFCDIRECTION((1.,0.,0.));
+#7=IFCDIRECTION((0.,0.,1.));
+#8=IFCCARTESIANPOINT((0.,0.,0.));
+#9=IFCAXIS2PLACEMENT3D(#8,#7,#6);
+#10=IFCDIRECTION((0.,1.));
+#11=IFCGEOMETRICREPRESENTATIONCONTEXT($,'Model',3,1.E-05,#9,#10);
+#12=IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#13=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#14=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#15=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#16=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#17=IFCMEASUREWITHUNIT(IFCPLANEANGLEMEASURE(0.017453292519943295),#16);
+#18=IFCCONVERSIONBASEDUNIT(#12,.PLANEANGLEUNIT.,'DEGREE',#17);
+#19=IFCUNITASSIGNMENT((#13,#14,#15,#18));
+#20=IFCPROJECT('20LlEYg5D1ERA7Bh_53DI_',#5,'',$,$,$,$,(#11),#19);
+#21=IFCSITE('0ie$bpOszCUgyIh5OXWXsG',#5,$,$,$,$,$,$,.ELEMENT.,$,$,$,$,$);
+#22=IFCRELAGGREGATES('2JPb$q1dPCpBGLb2SYvRVr',#5,$,$,#20,(#21));
+#23=IFCBUILDING('3qEEBpFm16UBTLeLB$yPTY',#5,$,$,$,$,#33,$,.ELEMENT.,$,$,$);
+#24=IFCRELAGGREGATES('3N6QCZZqr4xBkcDTiNPpEs',#5,$,$,#21,(#23));
+#25=IFCCARTESIANPOINT((0.,0.));
+#26=IFCAXIS2PLACEMENT2D(#25,$);
+#27=IFCRECTANGLEPROFILEDEF(.AREA.,$,#26,20000.,20000.);
+#28=IFCCARTESIANPOINT((0.,0.,0.));
+#29=IFCAXIS2PLACEMENT3D(#28,$,$);
+#30=IFCDIRECTION((0.,0.,1.));
+#31=IFCEXTRUDEDAREASOLID(#27,#29,#30,20000.);
```

Comment:
While I understand that this is only a testing file, I would still recommend using plausible values. 20x20x20 kilometres body for a building seems too much.